Casks/c/cables.rb

```diff
@@ -0,0 +1,31 @@
+cask "cables" do
+  arch arm: "-arm64", intel: "-x64"
+
+  version "0.2.0"
+  sha256 arm:   "3f78eb052b2e6ee47ba03b517b80fafa8c737e0d59dce6ebd888b87269e34731",
+         intel: "22104e4e8ba027aa5dcb063b616988c32acff3da5b273705798382b63a7d4a75"
+
+  url "https://github.com/cables-gl/cables_electron/releases/download/v#{version}/cables-#{version}-mac#{arch}.dmg"
+  name "Cables.gl"
```

Comment:
```suggestion
  name "Cables"
```

Let's also make the appropriate change here.